src/components/Popover/Popover.scss

```diff
@@ -52,7 +53,7 @@ $content-max-width: rem(400px);
 }
 
 .positionedAbove {
-  margin: spacing() 0 $visible-portion-of-arrow spacing(tight);
+  margin: spacing() spacing(tight) $visible-portion-of-arrow;
```

Comment:
Adjusting padding for Popovers with elements positioned above. This was a regression when we removed the small triangle from Popover.

@HYPD @sarahill this might need a Figma tweak?